src/Security/src/Authentication.CloudFoundry/ServiceCollectionExtensions.cs

```diff
@@ -21,16 +22,18 @@ public static class ServiceCollectionExtensions
     /// <param name="services">
     /// Service collection.
     /// </param>
-    public static void AddCloudFoundryContainerIdentity(this IServiceCollection services)
+    /// <param name="configuration">
+    /// The root <see cref="IConfiguration" /> to monitor for changes.
+    /// </param>
+    /// <param name="fileProvider">
+    /// Provides access to the file system.
+    /// </param>
+    public static void AddCloudFoundryContainerIdentity(this IServiceCollection services, IConfiguration configuration, IFileProvider fileProvider)
```

Comment:
Should we add a HostBuilder extension method that calls this and `IConfigurationBuilder.AddCertificate`, to set everything up in a single call?

Reply:
Good question, but how about if we revisit this in #1311 since there are more API changes there?